examples/utils/bundle-size-tests/webpack.config.js

```diff
@@ -8,6 +8,22 @@ const { BundleComparisonPlugin } = require("@mixer/webpack-bundle-compare/dist/p
 const { BundleAnalyzerPlugin } = require("webpack-bundle-analyzer");
 const DuplicatePackageCheckerPlugin = require("@cerner/duplicate-package-checker-webpack-plugin");
 const { BannedModulesPlugin } = require("@fluidframework/bundle-size-tools");
+const { fromInternalScheme, toInternalScheme } = require("@fluid-tools/version-tools");
+
+// We need to replace the version string in the bundled code; otherwise the bundle we build in CI for PRs will have the
```

Comment:
```suggestion
// We need to replace the version string in the bundled code (in the packageVersion.ts files); otherwise the bundle we build in CI for PRs will have the
```